pkg/controllers/routereflector/topologies/single.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2020 IBM Corporation All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package topologies
+
+import (
+	"fmt"
+	"math"
+	"sort"
+
+	apiv3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// SingleTopology single cluster topology with one ClusterID
+type SingleTopology struct {
+	Config
+}
+
+// IsRouteReflector if node has exact the same label and value
+func (t *SingleTopology) IsRouteReflector(_ string, labels map[string]string) bool {
```

Comment:
I'm not sure I understand why this varies between the single / multi topology. In the multi topology, this is almost the same logic except we don't check the value of the label? Could / should we ignore the label value here as well?

Reply:
This was because of the exported label configuration. In new version we would hard code the label and value, so this should be tidy up.